Tests/DefaultsTests/DefaultsTests.swift

```diff
@@ -2,6 +2,9 @@ import Foundation
 import XCTest
 import Defaults
 import CoreData
+#if canImport(Combine)
```

Comment:
I don't think this is necessary. People are expected to use the latest Xcode version.